ui/src/Components/SearchCandidates.jsx

```diff
@@ -37,63 +28,82 @@ export default class SearchCandidates extends React.Component {
 		.catch((error) => {console.log(error)});
 	}
 
-	/** Event Handlers for search fields
-		// TODO: Uncomment so state is updated, triggering a rerender of results
 	onNameSearch = (event) => {
-		this.setState({ searchName: event.target.value });
+		this.setState({ searchName: event.target.value.toLowerCase() });
+		console.log(event.target.value)
 		};
 
 	onEmailSearch = (event) => {
-		this.setState({ searchEmail: event.target.value });
+		this.setState({ searchEmail: event.target.value.toLowerCase() });
 		};
 
 	onNumberSearch = (event) => {
 		this.setState({ searchNumber: event.target.value });
 		};
 
-	*/
+	populate = (a, obj, i)=> {
+		a.push(<tr>
```

Comment:
not too sure what `a` is, but I would be in favour of having a more meaningful variable name to make the code self-explanatory for someone that needs to make changes in the future.

Reply:
Yup that makes sense!